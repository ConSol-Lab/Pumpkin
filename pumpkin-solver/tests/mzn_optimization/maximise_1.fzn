var 5..10: x :: output_var;
var -3..15: y :: output_var;
var 7..25: z :: output_var;
var -10..30: objective :: output_var;

constraint int_lin_eq([1, 1, 1], [x, y, z], 17);
constraint array_int_minimum(objective, [x, y, z]);

solve maximize objective;
