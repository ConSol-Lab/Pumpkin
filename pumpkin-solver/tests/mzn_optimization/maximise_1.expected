x = 6;
y = 2;
z = 9;
objective = 2;
----------
x = 5;
y = 5;
z = 7;
objective = 5;
----------
==========
